### PR TITLE
RET3109

### PR DESCRIPTION
**Pull request by addison-jephtah**
### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/BundlesRespondentController.java [hunk 1]
```diff
@@ -201,18 +204,20 @@ public ResponseEntity<CCDCallbackResponse> submitted(
...
         String header = "<h1>You have sent your hearing documents to the tribunal</h1>";
-        String body = """
-                <h2>What happens next</h2>
-                    The tribunal will let you know
-                    if they have any questions about the hearing documents you have submitted.
-                """;
-        response.setConfirmation_header(header);
-        response.setConfirmation_body(body);
-        return ResponseEntity.ok(response);
+        String body = "<h2>What happens next</h2>\r\n\r\nThe tribunal will let you know"
```
Comment (marcjudge) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/BundlesRespondentController.java:209]: https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=1350&id=et-ccd-callbacks&open=AYuRG_DO_Ip3et0Tw0Na

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -184,4 +184,6 @@ hmcts_service_id: BHA1
 launchdarkly:
   sdk-key: ${LAUNCH_DARKLY_SDK_KEY:sdk-key}
   offline-mode: false
-  env: ${LAUNCH_DARKLY_ENV:default}
\ No newline at end of file
+  env: ${LAUNCH_DARKLY_ENV:default}
+
+bundlesClaimantSubmittedRespondentNotificationTemplateId: ${BUNDLES_CLAIMANT_SUBMITTED_RESPONDENT_NOTIFICATION_TEMPLATE_ID:5c829358-0b54-4335-9d60-1c8296c4a516}
```
Comment (marcjudge) [inline src/main/resources/application.yaml:189]: ON L89 the template variables begin. Please put this variable in that block.

File: src/test/apiTest/resources/application.yaml [hunk 3]
```diff
@@ -133,4 +133,7 @@ hmcts_service_id: BHA1
 launchdarkly:
   sdk-key: ${LAUNCH_DARKLY_SDK_KEY:sdk-key}
   offline-mode: false
-  env: ${LAUNCH_DARKLY_ENV:default}
\ No newline at end of file
+  env: ${LAUNCH_DARKLY_ENV:default}
+
+bundlesClaimantSubmittedRespondentNotificationTemplateId: ${BUNDLES_CLAIMANT_SUBMITTED_RESPONDENT_NOTIFICATION_TEMPLATE_ID:5c829358-0b54-4335-9d60-1c8296c4a516}
```
Comment (marcjudge) [inline src/test/apiTest/resources/application.yaml:138]: Same as before. Move inside template block

File: src/test/integration/resources/application.yaml [hunk 4]
```diff
@@ -130,4 +130,6 @@ hmcts_service_id: BHA1
 
 launchdarkly:
   sdk-key: ${LAUNCH_DARKLY_SDK_KEY:sdk-key}
-  env: ${LAUNCH_DARKLY_ENV:default}
\ No newline at end of file
+  env: ${LAUNCH_DARKLY_ENV:default}
+
+bundlesClaimantSubmittedRespondentNotificationTemplateId: ${BUNDLES_CLAIMANT_SUBMITTED_RESPONDENT_NOTIFICATION_TEMPLATE_ID:5c829358-0b54-4335-9d60-1c8296c4a516}
```
Comment (marcjudge) [inline src/test/integration/resources/application.yaml:135]: Same as before. Move inside template block

**Review by marcjudge — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yaml (hunk 2 above)
Comment (addison-jephtah) [reply src/main/resources/application.yaml]: The variable is moved within the template block

File: src/test/integration/resources/application.yaml (hunk 4 above)
Comment (addison-jephtah) [reply src/test/integration/resources/application.yaml:135]: The variable is moved within the template block

File: src/test/apiTest/resources/application.yaml (hunk 3 above)
Comment (addison-jephtah) [reply src/test/apiTest/resources/application.yaml]: The variable is moved within the template block

**Review by marcjudge — APPROVED** [review]

**Review by jip1984 — APPROVED** [review]

**Pull request merged**